### ISSUE TITLE
[SUGGESTION] defining keys&salts in .env file

### ISSUE THREAD

**Issue by Jaspervv** [themosis/themosis#265]
WP Salts&Keys should be unique on every instance of your site. And they should be kept private.
Currently they're part of one's git repo tho because they're defined inside the `config/wordpress.php`:
```php
// config/wordpress.php:22
define('AUTH_KEY', 'put your unique phrase here');
define('SECURE_AUTH_KEY', 'put your unique phrase here');
define('LOGGED_IN_KEY', 'put your unique phrase here');
define('NONCE_KEY', 'put your unique phrase here');
define('AUTH_SALT', 'put your unique phrase here');
define('SECURE_AUTH_SALT', 'put your unique phrase here');
define('LOGGED_IN_SALT', 'put your unique phrase here');
define('NONCE_SALT', 'put your unique phrase here');
```

Wouldn't it be better to define them in the .`env`, allowing each instance of the site to have their own, and removing secure information from the git repo?
```php
// config/wordpress.php:22
define('AUTH_KEY', env('AUTH_KEY', 'unique phrase') );
define('SECURE_AUTH_KEY', env('SECURE_AUTH_KEY', 'unique phrase') );
define('LOGGED_IN_KEY', env('LOGGED_IN_KEY', 'unique phrase') );
define('NONCE_KEY', env('NONCE_KEY', 'unique phrase') );
define('AUTH_SALT', env('AUTH_SALT', 'unique phrase') );
define('SECURE_AUTH_SALT', env('SECURE_AUTH_SALT', 'unique phrase') );
define('LOGGED_IN_SALT', env('LOGGED_IN_SALT', 'unique phrase') );
define('NONCE_SALT', env('NONCE_SALT', 'unique phrase') );
```

**Comment by jlambe** [comment]
@Jaspervv Yes indeed it should be but I kept it there for now because it's a simple copy/paste from the key generator provided by WordPress.

I would like to move them on the `.env` file but for this, I would like to also provide a command to generate them automatically for the user so we don't have to manually set them up one by one. That command can be targeted on a `2.1` release and could secure more the installation.

**Comment by Jaspervv** [comment]
We currently have commands for that in our Themosis plugin, I'll make a PR but I don't think they're up to standards. You might like it as an example though.

**Comment by KevinBatdorf** [comment]
I just saw this but made a PR to comment them out. A command would be nice or just instructions to use `wp config shuffle-salts`. I suppose better to have them in .env

Maybe comment them out for 2.0 release? Leaving them uncommented will break wp-cli usage.

**Comment by jlambe** [comment]
This can be planned on a `2.1` release. I don't think this is going to break an update as current site may continue by keeping salt keys inside the `wordpress.php` configuration and new installation will use the new command and generate them inside the `.env` upon root installation.
